src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/Dataflow/DiagnosticUtilities.cs

```diff
@@ -79,6 +82,21 @@ internal static bool TryGetRequiresAttribute(TypeSystemEntity member, string req
             return true;
         }
 
+        public static CustomAttributeValue<TypeDesc>? GetDecodedCustomAttribute(this PropertyPseudoDesc This,
```

Comment:
Nit: I find the `This` name pretty weird... would go with just `prop` or something similar.